Tasks/NugetPublisher/task.json

```diff
@@ -9,7 +9,7 @@
     "version": {
         "Major": 0,
         "Minor": 2,
-        "Patch": 7
+        "Patch": 8
```

Comment:
Could you actually bump this to 0.2.9? We shipped a customer a private build marked 0.2.8